clang/docs/LanguageExtensions.rst

```diff
@@ -3379,6 +3379,60 @@ Query for this feature with ``__has_builtin(__builtin_debugtrap)``.
 
 Query for this feature with ``__has_builtin(__builtin_trap)``.
 
+``__builtin_verbose_trap``
```

Comment:
@ahatanak Is this in the wrong place in the document? `__builtin_verbose_trap` is lexicographically after `__builtin_nondeterministic_value`

Reply:
It doesn't look like the existing builtins are in lexicographical order. `__builtin_debugtrap` is followed by `__builtin_trap`, which is followed by `__builtin_nondeterministic_value`.

Reply:
Ah, OK - that seems like more reason the prefix should be singular. Like we shouldn't add a new/distinct prefix for array bounds? Or other things we want to error out on - because it'd be an ongoing maintenance problem updating consumers like lldb each time we add one? 

So maybe we should have a generic prefix we'll always use (`__builtin_trap`, say) and then maybe some identifier after that in a known format (eg: space, identifier, space) that can be used? then an error message string?